Anytype/Sources/PresentationLayer/Flows/SpaceSettings/SpaceSettingsCoordinatorViewModel.swift

```diff
@@ -102,14 +106,26 @@ final class SpaceSettingsCoordinatorViewModel: ObservableObject, SpaceSettingsMo
     // MARK: - PersonalizationModuleOutput
     
     func onDefaultTypeSelected() {
-        let module = newSearchModuleAssembly.objectTypeSearchModule(
-            title: Loc.chooseDefaultObjectType,
-            spaceId: activeWorkspaceStorage.workspaceInfo.accountSpaceId
-        ) { [weak self] type in
-            self?.objectTypeProvider.setDefaultObjectType(type: type, spaceId: type.spaceId)
-            self?.navigationContext.dismissTopPresented(animated: true)
+        if FeatureFlags.newTypePicker {
+            let module = objectTypeSearchModuleAssembly.objectTypeSearchForCreateObject(
```

Comment:
Method is called `objectTypeSearchForCreateObject`. Rename it to default method for assemblies `make`